corehq/apps/app_manager/tests/data/suite/reports_module_summary_detail_use_localized_description.xml

```diff
@@ -31,7 +31,7 @@
         </header>
         <template>
           <text>
-            <xpath function="report description"/>
+            <xpath function='description'/>
```

Comment:
I'm not following the test changes: if this file is using a localized description, shouldn't it use a `locale` element for the description, not an `xpath` element?